src/style.css

```
@@ -0,0 +1,63 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  max-width: 1200px;
+  height: 60px;
+  display: flex;
+  margin: auto;
+
+}
+
+.logo {
+  flex-grow: 1;
+}
+
+.logo-img {
+  margin: 10px 0 10px 50px;
+}
+
+.nav {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  align-items: center;
+  position: relative;
+  padding: 0 50px 0 0;
+}
+
+.nav-link {
+  margin-left: 20px;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: 14px;
+  color: #000;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 37px;
```

```suggestion
  width: 100%;
```
якщо так записати то ширина лінії буде підходити для усіх слів

---

если поставить 100% то полоса растягивается на все ссылки. нужно добавлять еще селекторов. но сам смысл понял.